modules/alma/lib/AlmaClient/AlmaClient.class.php

```diff
@@ -980,6 +988,39 @@ public function remove_message_service($borr_card, $pin_code, $method, $type) {
     return TRUE;
   }
 
+  /**
+   * Create new user at alma.
+   *
+   * @param $cpr
+   *   The users CPR number.
+   * @param $pin_code
+   *   The users pin-code.
+   * @param $name
+   *   The users full name.
+   * @param $mail
+   *   The users e-mail address.
+   * @param $branch
+   *   The users preferred pick-up branch.
+   *
+   * @return \DOMDocument
+   *   The raw XML response from the library system.
```

Comment:
This seems very crude.

A raw XML response should not leave the client.  As far as I can tell the return value is not used in the current form, but we do not want anyone to actually use it in that form either.

I would prefer that we do not return anything - alterantively `TRUE`.